release/models/wifi/openconfig-wifi-mac.yang

```diff
@@ -26,7 +26,13 @@ module openconfig-wifi-mac {
   description
     "Model for managing MAC layer configuration of Radio interfaces.";
 
-  oc-ext:openconfig-version "1.0.0";
+  oc-ext:openconfig-version "1.1.0";
```

Comment:
I think we also need to update the following:
* "leaf frequency" line 1373 description.
* leaf operating_frequency" line 123

Reply:
Fixed the descriptions of both leaves.